src/components/FormComponents/DiagnosePicker/DiagnosePicker.tsx

```diff
@@ -122,7 +152,7 @@ function DiagnoseDescription({
     return (
         <div className={cn(className, 'navds-form-field navds-form-field--medium')}>
             <Label>{label}</Label>
-            <BodyLong className={styles.diagnoseDescriptionText}>{text ?? '-'}</BodyLong>
+            <BodyShort className={cn('h-12 flex items-start')}>{text ?? '-'}</BodyShort>
```

Comment:
Kan fjerne `cn` her?

```suggestion
            <BodyShort className={'h-12 flex items-start'}>{text ?? '-'}</BodyShort>
```